spec/workers/enqueue_subject_queue_worker_spec.rb

```diff
@@ -53,23 +53,135 @@
       end
     end
 
-    describe "#load_subjects" do
+    context "when selecting via strategy param" do
+      let(:result_ids) { [1] }
+
+      it 'should fallback from the override strategy' do
+        allow_any_instance_of(Subjects::PostgresqlSelection).to receive(:select).and_return(result_ids)
+        expect(SubjectQueue).to receive(:enqueue)
+        subject.perform(workflow.id, nil, nil, nil, :unknown_strategy)
+      end
+
+      context "with cellect" do
+        let(:run_selection) { subject.perform(workflow.id, nil, nil, nil, :cellect) }
+        before do
+          allow(Panoptes).to receive(:cellect_on).and_return(true)
+        end
+
+        it 'should use the cellect client' do
+          expect(Subjects::CellectClient).to receive(:get_subjects).and_return(result_ids)
+          run_selection
+        end
+
+        it 'should attempt to queue the selected set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.